flujo/cli/lock_command.py

```
@@ -0,0 +1,576 @@
+    # Version defaults to "0.1" (YAML default)
+    pipeline_version: str = getattr(pipeline, "version", "0.1")
+
+    # Generate a stable ID based on pipeline name
+    pipeline_id: str = str(uuid.uuid5(uuid.NAMESPACE_DNS, pipeline_name))
+
+    return pipeline_name, pipeline_version, pipeline_id
+
+
+def _resolve_external_files(
+    external_files: Optional[List[str]],
+    project_root: Path,
```

Ruff (0.14.10)

Unused function argument: `project_root`

(ARG001)